fluff/app_config.py

```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+from django.db.models.signals import post_migrate
+from fluff.signals import catch_signal
+
+
+class FluffAppConfig(AppConfig):
+    name = 'fluff'
+
+    def ready(self):
+        post_migrate.connect(catch_signal, sender=self)
```

Comment:
I don't think so but could be good practice to do it anyway.

On 19 October 2015 at 19:46, Ethan Soergel <notifications@github.com> wrote:

> In fluff/app_config.py
> <https://github.com/dimagi/fluff/pull/105#discussion_r42402361>:
>
> > @@ -0,0 +1,10 @@
> > +from django.apps import AppConfig
> > +from django.db.models.signals import post_migrate
> > +from fluff.signals import catch_signal
> > +
> > +
> > +class FluffAppConfig(AppConfig):
> > +    name = 'fluff'
> > +
> > +    def ready(self):
> > +        post_migrate.connect(catch_signal, sender=self)
>
> Curious if you think we need to worry about this
> <https://docs.djangoproject.com/en/1.8/topics/signals/#connecting-receiver-functions>
> :
>
> The ready() method may be executed more than once during testing, so you
> may want to guard your signals from duplication
> <https://docs.djangoproject.com/en/1.8/topics/signals/#preventing-duplicate-signals>,
> especially if you’re planning to send them within tests.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/fluff/pull/105/files#r42402361>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa